cmd/worker/worker.go

```diff
@@ -60,8 +65,10 @@ func (app *App) HandleAttachToTangle(ctx context.Context, j *job.IRIJob) {
 			app.failJob(j, err.Error())
 			return
 		}
+		*/
 
-		outTrytes = append(outTrytes, string(out))
+		out := C.ccurl_pow(C.CString(ts), C.int(j.AttachToTangleRequest.MinWeightMagnitude))
```

Comment:
`CString`s need to be `free`d again because they're not garbage collected automatically. 

```go
cTrytes := C.CString(ts)
...
C.free(unsafe.Pointer(cTrytes))
```